doc/source/overview/high_level.md

```diff
@@ -0,0 +1,1086 @@
+  and [random forests](https://en.wikipedia.org/wiki/Random_forest).
+
+### Applications
+
+As machine learning methods have become more complex and more mainstream, with many industries
+now [incorporating AI](https://www.mckinsey.com/business-functions/mckinsey-analytics/our-insights/global-survey-the-state-of-ai-in-2020)
+in some form or another, the need to understand the decisions made by models is only increasing. Explainability has
+several applications of importance.
+
+- **Trust:** At a core level, explainability builds [trust](https://onlinelibrary.wiley.com/doi/abs/10.1002/bdm.542) in
+  the machine learning systems we use. It allows us to justify their use in many contexts where an understanding of the
+  basis of the decision is paramount. This is a common issue within machine learning in medicine, where acting on a
+  model prediction may require expensive or risky procedures to be carried out.
+- **Testing:** Explainability might be used to [audit financial models](https://arxiv.org/abs/1909.06342) that aid
+  decisions about whether to grant customer loans. By computing the attribution of each feature towards the prediction
+  the model makes, organisations can check that they are consistent with human decision-making. Similarly,
+  explainability applied to a model trained on image data can explicitly show the model's focus when making decisions,
+  aiding [debugging](http://proceedings.mlr.press/v70/sundararajan17a.html). Practitioners must be wary
+  of [misuse](#biases), however.
+- **Functionality:** Insights can be used to augment model functionality. For instance, providing information on top of
+  model predictions such as how to change model inputs to obtain desired outputs.
+- **Research:** Explainability allows researchers to understand how and why opaque models make decisions. This can help
+  them understand more broadly the effects of the particular model or training schema they're using.
+
+### Black-box vs White-box methods
+
+Some explainers apply only to specific types of models such as the [Tree SHAP](path-dependent-tree-shap) methods which
+can only be used with [tree-based models](https://en.wikipedia.org/wiki/Decision_tree_learning). This is the case when
+an explainer uses some aspect of that model's internal structure. If the model is a neural network then some methods
+require taking gradients of the model predictions with respect to the inputs. Methods that require access to the model
+internals are known as **white-box** methods. Other explainers apply to any type of model. They can do so because the
+underlying method doesn't make use of the model internals. Instead, they only need to have access to the model outputs
+given particular inputs. Methods that apply in this general setting are known as **black-box** methods. Note that
+white-box methods are a subset of black-box methods and an explainer being a white-box method is a much stronger
```

Comment:
This statement is a bit weird: "Note that white-box methods are a subset of black-box methods and an explainer being a white-box method is a much stronger constraint than it being a black-box".
a) Should be the set of models that a black-box method applies to is a superset of the set of models a white-box method applies to.
b) black box methods apply to all models so it's sort of trivial to say white-box subset of black-box